ko-KR/index.html

```diff
@@ -0,0 +1,256 @@
+    </div>
+
+    <div class='row'>
+      <div class='col-sm-12'>
+        <h2 class='header'>참여</h2>
+
+        <p id='contributing'>
+          Servo 프로젝트는 경험이 풍부한 개발자부터 새로운 개발자 모두의 참여를 장려합니다.
+
+          페이지 하단의 링크를 통하여 더 자세한 정보를 얻을 수 있으며
+          Servo를 더욱 좋게 만들기 위하여 전 세계에서 일하고 있는 수백 명의 참여자가 함께합니다.
+        </p>
+
+        <p>
+          Servo에 대한 참여나 그에 따른 의존성은 GitHub의 Pull Request를 통하여 이루어져야 합니다.
+          각 Pull Request는 핵심 참여자들이 (패치 적용의 권한이 있는 사람) 주 트리에 패치를 하거나
+          필요한 변경 사항에 대한 피드백을 받습니다.
+          모든 참여는 정해진 이 Pull Request 형식을 따라 해야 , 심지어 핵심 참여자들도 따라야 합니다.
+        </p>
+
+        <p>
+          어떤 문제를 해결하고 싶으시다면, 먼저 GitHub issue에 의견을 제시해주세요.
+          이는 같은 문제에 대한 참여자들의 중복된 노력을 방지하기 위한 것입니다.
+        </p>
+
+        <p>
+          만약 쉬운 버그를 찾고 있으시다면 GitHub의 
+          <a href="https://github.com/servo/servo/labels/E-easy">E-Easy issue 태그</a>를
+          살펴보세요.
+        </p>
+
+        <p>
+          Servo에 관한 작업을 시작하는 방법에 대해서는
+          <a href="https://github.com/servo/servo/blob/master/docs/HACKING_QUICKSTART.md">HACKING_QUICKSTART.md</a>
+          를 참고해주세요.
+        </p>
+
+        <h3>Pull Request 체크리스트</h3>
+
+        <ul>
+          <li>
+            master 브랜치에서 새로 브랜치를 생성하시고,
+            필요한 경우 Pull Request 전에 현재 master 브랜치로 rebase 해주세요.
+            master와 완전히 병합되지 않는 경우 변경 내용을 다시 적용하라는 메시지가 표시될 수 있습니다.
+          </li>
+
+          <li>
+            서브 모듈 업데이트가 패치 된 커밋이 아닌 한 Pull Request를 하지 말아주세요.
+          </li>
+
+          <li>
+            패치를 검토하지 않거나, 검토할 특정 사용자가 필요한 경우,
+            검토자에게 Pull Request이나 comment를 요청하거나 irc.mozilla.org의 #servo 채널에서 검토를 요청할 수 있습니다.
+          </li>
+
+          <li>
+            수정된 버그 또는 새로운 기능과 관련된 테스트를 추가해주세요.
+            DOM 변경의 경우 대게 웹 플랫폼 테스트 (레이아웃, reftest) 입니다.
+            자세한 내용은
+            <a href="https://github.com/servo/servo/wiki/Testing">테스트 가이드</a>
+            를 참고해주세요.
+          </li>
+
+          <li>
+            자세한 git 명령어들은
+            <a href="https://github.com/servo/servo/wiki/Github-workflow">GitHub 워크플로우</a>
+            를 참고해주세요.
+          </li>
+        </ul>
+
+        <a class='btn btn-primary btn-large' role='button'
+        href='https://starters.servo.org/'>참여 시작하기</a>
+
+      </div>
+    </div>
+
+    <div class='row'>
+      <div class='col-sm-12'>
+        <h2 class='header'>커뮤니케이션</h2>
+
+        <p>
+          Servo의 참여자들은 
+          <a href="https://wiki.mozilla.org/IRC">irc.mozilla.org</a> 의 #servo 채널을 빈번히 사용합니다.
+          <a href="https://lists.mozilla.org/listinfo/dev-servo">dev-servo 메일링 리스트</a>에 가입할 수 있습니다.
+        </p>
+
+      </div>
+    </div>
+
+    <div class='row'>
+      <div class='col-sm-12'>
+        <h2 class='header'>행동 규칙</h2>
+
+        <p>
+          우리는 
+          <a href="https://www.rust-lang.org/conduct.html">Rust 행동 규칙</a>
+          을 따릅니다.
+        </p>
+
+      </div>
+    </div>
+
+    <div class='row' id='languages'>
+      <div class='col-sm-12'>
+
+        <p>다른 언어의 페이지:
```

Comment:
Other languages pages need to add Korean.

Reply:
That exist 231 line. https://github.com/servo/servo.org/pull/54/files#diff-ea3f63884f0b3f27a41833c393aebea4R231

Reply:
I'm sorry, I misunderstand at first. I edited it.